volatility3/framework/plugins/windows/handles.py

```diff
@@ -349,6 +355,8 @@ def _generator(self, procs):
             symbol_table=kernel.symbol_table_name,
         )
 
+        object_types = [s.lower() for s in self.config.get("type", [])]
```

Comment:
Don't forget to update this to `types` too.

Reply:
Thanks, fixed here: 7ef1aa6408bbe67e0a36386ec89788e65feff65a